### ISSUE TITLE
Error updating Kaizen Editor to version 0.8.0.201909121743

### ISSUE THREAD

**Issue by cvgaviao** [RepreZen/KaiZen-OpenAPI-Editor#520]
Eclipse just popup an update window and I authorized it to run...
Then this error message was showed up:
```
An error occurred while collecting items to be installed
session context was:(profile=_home_cvgaviao_opt_eclipse-ide_java-201906_eclipse, phase=org.eclipse.equinox.internal.p2.engine.phases.Collect, operand=, action=).
Problems downloading artifact: org.eclipse.update.feature,com.reprezen.swagedit.feature,0.8.0.201909121743.
SHA-256 hash is not as expected. Expected: fa0877e1fe5cae38f20bab4e02c410fddc644448c56a9e7d498f64064701b181 and found b76ce715bee11b3d9060fc3288938f470ae246d6fdc9eca88e009f18e377d78c.
```

**Comment by tedepstein** [comment]
@cvgaviao , I'm seeing similar errors when trying to install into Eclipse 2019-06: 

```
An error occurred while collecting items to be installed
session context was:(profile=C__Users_tedep_RepreZen_API_Studio_eclipse-install-test_clean_jee-2019-06_eclipse, phase=org.eclipse.equinox.internal.p2.engine.phases.Collect, operand=, action=).
Problems downloading artifact: osgi.bundle,com.reprezen.swagedit,0.8.0.201909121743.
SHA-256 hash is not as expected. Expected: 737296a9978a442291fbe979c5a8851efe625623fa936fcd9cc09baac8ca0cae and found 79fcf7ee0ba1961f1e8fdfe2567d6283b5a7ed3e21123b94c6596b451e4cf391.
Problems downloading artifact: osgi.bundle,com.reprezen.swagedit.core,0.8.0.201909121743.
SHA-256 hash is not as expected. Expected: ec416696a03390e6dd22f2f90a6317864ac0e5aa8d52f8ba38af3a6338296a9e and found 532cdcc7ae0e984ba448ca05ca2bf2c903a5d77fcc69e0ce6123086e21b266a3.
Problems downloading artifact: osgi.bundle,com.reprezen.swagedit.dependencies,0.8.0.201909121743.
SHA-256 hash is not as expected. Expected: acf6e157d675502d6b157066aaa2d974f30d721b4574bcbf630ce9cf1b8beb0e and found a224b7721fd176f2cc960466606f61da2a97a5cbc66fd6997ccc56f5861a7f2e.
Problems downloading artifact: org.eclipse.update.feature,com.reprezen.swagedit.feature,0.8.0.201909121743.
SHA-256 hash is not as expected. Expected: fa0877e1fe5cae38f20bab4e02c410fddc644448c56a9e7d498f64064701b181 and found b76ce715bee11b3d9060fc3288938f470ae246d6fdc9eca88e009f18e377d78c.
Problems downloading artifact: osgi.bundle,com.reprezen.swagedit.openapi3,0.8.0.201909121743.
SHA-256 hash is not as expected. Expected: 7aa5c35d54b6db661e8e43f7f80f63e29b9c6fcc1ab9a149e07aa7111c68fd1a and found 5054f15f4f55fa5376b68a264e9c06ba825a28752b6c70005ba2eda7023910d6.
```

Looking into this...

**Comment by tedepstein** [comment]
@cvgaviao , thanks for letting us know about this. 

I have rolled back the recent release, so Eclipse Marketplace will now install KaiZen-OpenAPI Editor 0.8.0.201907182101. We may need a bit more time to determine the cause and remediate the problem with the last release build. As soon as we get it fixed, we'll do a new release. 

In the meantime, I'd recommend uninstalling KZOE, and re-installing from Marketplace.

**Comment by tedepstein** [comment]
@cvgaviao , this is now fixed, and we have a new KZOE release 0.8.0.201909171245 available on Eclipse Marketplace. 

Thanks for your help & patience!
